integrators/cohen_hubbard_oesterwinter.hpp

```diff
@@ -0,0 +1,34 @@
+﻿
+#pragma once
+
+#include "numerics/fixed_arrays.hpp"
+
+namespace principia {
+namespace integrators {
+namespace internal_cohen_hubbard_oesterwinter {
+
+using numerics::FixedVector;
+
+// Definition of a Cohen-Hubbard-Oesterwinter formula.  |order_| is the order of
+// the approximation, that is, the error on the derivative is h^(order + 1).
+// The formula requires |order_| values of the acceleration.  numerators[0]
```

Comment:
Document what the formula is, and what it approximates.

Reply:
Done.